staging/src/k8s.io/apiserver/pkg/authentication/request/x509/x509_test.go

```diff
@@ -37,9 +37,31 @@ const (
 
 	/*
 
+	   > root key
+	   openssl genrsa -out root.key 1024 && \
+	   cat root.key
+
+	   > output
+
+	   -----BEGIN PRIVATE KEY-----
```

Comment:
Its obvious to us, but put a note here that this is for testing only. K8s gets all kinds of hacker reports for test keys and they seem to more often ignore if there's a comment immediately preceding the key